013-quarkus-oidc-restclient/src/main/java/io/quarkus/qe/ping/clients/TokenPropagationPongClient.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.qe.ping.clients;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterRestClient
+// Can't use both OIDC Client Filter and OIDC Token extensions. Reported in https://github.com/quarkusio/quarkus/issues/14466
```

Comment:
Again, the TODO would be really nice here

Reply:
added